samples/sqltransport-sqlpersistence/SqlServer_3/Sender/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NServiceBus.Persistence.Sql;
+
+#region SqlPersistenceSettings
+
+[assembly: SqlPersistenceSettings(
```

Comment:
As this sample targets SQL Server, why does it have flags for SQL Server and MySql?
(Ref: https://docs.particular.net/nservicebus/sql-persistence/#script-creation-to-produce-only-ms-sql-server-scripts)